src/runtime/auto-prompt-manager.js

```diff
@@ -44,6 +46,23 @@ const dismissEvents = [
   AnalyticsEvent.ACTION_CONTRIBUTION_OFFERS_CLOSED,
   AnalyticsEvent.ACTION_SUBSCRIPTION_OFFERS_CLOSED,
 ];
+/** @const {!Array<!AnalyticsEvent>} */
+const completedActions = [AnalyticsEvent.ACTION_SURVEY_DATA_TRANSFER];
+
+/**
+ * Returns storage key for a given event.
+ * @param {!AnalyticsEvent} event
+ * @return {string}
+ * @protected
+ */
+function completedActionToStorageKey_(event) {
+  switch (event) {
+    case AnalyticsEvent.ACTION_SURVEY_DATA_TRANSFER:
+      return STORAGE_KEY_SURVEY_COMPLETED;
+    default:
+      return 'unknownaction';
```

Comment:
Would it be possible to cover this line with a test?

Reply:
Refactored function into a `Map`, so this is no longer applicable. Marking resolved.